### PR TITLE
Remove remaining containerd fuzz and clean up build.sh

### PR DESCRIPTION
**Pull request by djdongjin**
Fix https://github.com/containerd/containerd/issues/7066
Equivalent fuzz in upstream: https://github.com/containerd/containerd/blob/main/contrib/fuzz/containerd_import_fuzzer.go

Signed-off-by: Jin Dong <djdongjin95@gmail.com>

### CODE REVIEW

File: projects/containerd/build.sh [hunk 1]
```diff
@@ -18,30 +18,16 @@ go get github.com/AdaLogics/go-fuzz-headers
...
-mv $SRC/cncf-fuzzing/projects/containerd/compression_fuzzer.go $SRC/containerd/archive/compression/
 compile_go_fuzzer github.com/containerd/containerd/archive/compression FuzzDecompressStream fuzz_decompress_stream
 
-mv $SRC/cncf-fuzzing/projects/containerd/*.go $SRC/containerd/contrib/fuzz/
 compile_go_fuzzer github.com/containerd/containerd/contrib/fuzz FuzzImagesCheck fuzz_images_check
 compile_go_fuzzer github.com/containerd/containerd/contrib/fuzz FuzzExchange fuzz_diff_compare
 compile_go_fuzzer github.com/containerd/containerd/contrib/fuzz FuzzDiffCompare fuzz_diff_compare
 compile_go_fuzzer github.com/containerd/containerd/contrib/fuzz FuzzDiffApply fuzz_diff_apply
 compile_go_fuzzer github.com/containerd/containerd/contrib/fuzz FuzzUUIDParse fuzz_uuid_parse
+compile_go_fuzzer github.com/containerd/containerd/contrib/fuzz FuzzContainerdImport fuzz_containerd_import
```
Comment (djdongjin) [inline projects/containerd/build.sh:33]: Not sure if this fuzz is already in good shape to be enabled.

**Comment by AdamKorcz** [comment]
Thank you. LGTM. Could you change "Romve" to "Remove" in the title?

File: projects/containerd/build.sh (hunk 1 above)
Comment (djdongjin) [reply projects/containerd/build.sh:33]: @kzys can you please confirm if this fuzz should be enabled now? Thanks!

**Comment by kzys** [comment]
Do we need to fix this script? oss-fuzz's containerd Dockerfile no longer uses cncf-fuzzing.

https://github.com/google/oss-fuzz/blob/master/projects/containerd/Dockerfile

**Comment by djdongjin** [comment]
https://github.com/google/oss-fuzz/blob/master/projects/containerd/build.sh

Oh right... just saw the cncf-fuzzing build.sh is removed from oss-fuzz's build.sh. :) I must have seen some stale commit...

In that case, should I just remove both the remaining fuzzer and this build.sh, (or the whole containerd folder)?

Thanks.

**Comment by kzys** [comment]
I'm planning to port https://github.com/cncf/cncf-fuzzing/blob/main/projects/containerd/containerd_import_structured_fuzzer.go to containerd, without bringing the dependencies.

**Comment by AdamKorcz** [comment]
> Do we need to fix this script? oss-fuzz's containerd Dockerfile no longer uses cncf-fuzzing.
> 
> https://github.com/google/oss-fuzz/blob/master/projects/containerd/Dockerfile

We can remove the whole folder instead like @djdongjin suggested.

**Comment by djdongjin** [comment]
@kzys I might miss something but does this one bring new dependency to containerd? Also, it seems identical with [containerd_import_fuzzer.go](https://github.com/containerd/containerd/blob/main/contrib/fuzz/containerd_import_fuzzer.go) in containerd except some syntax.

Thanks!

**Comment by AdamKorcz** [comment]
Is this one ready?

**Pull request closed without merging**